brain-marks/Localization/hi.lproj/Localizable.strings

```diff
@@ -0,0 +1,55 @@
+/*
+  Localizable.strings
+  brain-marks
+
+  Created by AC Richter on 13.12.21.
+  
+*/
+
+// TabView
+"Tweets" = "ट्वीट्स";
+"Settings" = "सेटिंग";
+
+// CategoryViews
+"Categories" = "श्रेणियाँ";
+"CategoriesAreEmpty" = "श्रेणियां खाली हैं, ";
+"PleaseAddNewCategories" = "कृपया क्लिक करके नई श्रेणियां जोड़ें";
+"YouHaventCreatedCategories" = "आपने कोई श्रेणी नहीं बनाई है!";
+"Edit" = "Edit";
+"Done" = "Done";
```

Comment:
I'm interested to know why `Edit` and `Done` aren't translated?

Reply:
Some words in hindi just don't have straight-up translations.